### PR TITLE
removed unnecessary clutter …

### PR DESCRIPTION
**Pull request by James9446**
such as old code that was commented out and a few component files that I do not need
@Chazie9 @openmichael @ChaseWNorton

### CODE REVIEW

**Review by Chazie9 — APPROVED** [review]
the deletions look fine. Only thing I did see is that you are not importing PledgeInfo to the client/src/index.jsx.  You didn't have it before, but I think you eventually will. I have a question though, since this comment does not really pertain to the cleanup you made, but I think will need to be done is this still the best place to mention it?

**Comment by James9446** [comment]
@Chazie9,
That import was there already. It is weird that it doesn't look like a normal line of code on here. Maybe it has something to do with having not been altered but you can see it greyed out with some @@ symbols in front of it. I just switched back to that branch and the client/src/index.jsx is importing PledgeInfo. but thank you for pointing that out though. I will definitely double check if its still there after I merge. Do I have approval? 

I guess you can just slack me you're question

**Comment by Chazie9** [comment]
@James9446 ok I didnt see it, yes approved

**Pull request merged**
